### ISSUE TITLE
[BUG] Items titles does not line break at 200% zoom

### ISSUE THREAD

**Issue by patrkjr** [kirbydesign/designsystem#4536]
### Describe the bug and what you expected to happen

An item title that contains too much text, should be able to use 2 lines, but does not in this case

### Describe how to reproduce the bug

Set text zoom to max (200%). Go to housing and press the loan of a customer. The resulting screen will contain an item with "Renteudgifter efter ...". 

### Severity

(2) High — feature broken

### Which Kirby version was used?

11.9.0-rc.1

### Add screenshots or videos

<img width="640" height="1385" alt="Image" src="https://github.com/user-attachments/assets/2436217c-e8a3-4b77-b20c-91ef2a69c338" />

### Environment (OS / Browser)

iOS 26

**Comment by Loujuu** [comment]
Item title need to be able to wrap to 2 lines and then truncate with resize over 145 procent or above.

**Comment by Loujuu** [comment]
In cookbook item title already wraps and truncates after two lines on text resize: @patrkjr Can you check if you have used                       `<p class="kirby-item-title">`

<img width="405" height="678" alt="Image" src="https://github.com/user-attachments/assets/1118a83f-8ed5-40f1-9f42-deb3ef0f3f18" />

**Comment by patrkjr** [comment]
Im being told that we dont by nim (Nikolaj Mols).

**Comment by Loujuu** [comment]
@patrkjr Okay, if you use the class it will wrap to two lines and truncate on text resize. I will close this task.
